src/assests/doctors.ts

```diff
@@ -0,0 +1,305 @@
+import { DoctorModel } from "@/models/doctor";
+
+import sara from "@/assests/doctors-images/sara.jpg";
+import john from "@/assests/doctors-images/john.jpg";
+import mary from "@/assests/doctors-images/mary.jpg";
+import jack from "@/assests/doctors-images/jack.png";
+import mateo from "@/assests/doctors-images/mateo.jpg";
+import martin from "@/assests/doctors-images/martin.jpg";
+import amaris from "@/assests/doctors-images/amaris.webp";
+import lukas from "@/assests/doctors-images/lukas.webp";
+import whitley from "@/assests/doctors-images/whitley.jpeg";
+import brody from "@/assests/doctors-images/brody.webp";
+
+import { SERVICE_TYPE } from "@/options/service-types-options";
+import { APPOINTMENT_TIMES } from "@/options/appointment-time-options";
+import { SPECIALITY_DEGREE } from "@/options/degree-options";
+
+export const doctors: DoctorModel[] = [
+  {
+    id: "97420f0d-b576-4f65-9ffc-a81b4b1b4e77",
+    name: "Sara Smith",
+    serviceType: SERVICE_TYPE[0].value,
+    serviceTypeName: SERVICE_TYPE[0].label,
```

Comment:
خب وقتی هم به value و هم به label احتیاج دارید، چرا جدا جدا پاس میدید؟

```suggestion
    serviceType: SERVICE_TYPE[0],
```